run-go-vet.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
+
 set -e
 pkg=$(go list)
-for dir in $(echo $@|xargs -n1 dirname|sort -u); do
-  go vet $pkg/$dir
+for fn in "$@"; do
+  go vet "${pkg}/${fn}"
```

Comment:
Does this work? `go help vet` claims it runs against packages not files.

My suggestion [here](https://github.com/dnephin/pre-commit-golang/pull/47#issuecomment-695854564) was something like this:

```shell
for dir in $(echo $@|xargs -n1 dirname|sort -u); do
    go vet $(go list ./$dir)
done
```

Reply:
Yup, with go 1.13, you can run it against files. Here's a test:

```shell
keda on 🌲 http Go 🐹  v1.13.8 go vet main.go

keda on 🌲 http Go 🐹  v1.13.8   echo $?
0
```